accounting/src/main/java/io/skai/accounting/service/ClientService.java

```diff
@@ -1,11 +1,12 @@
 package io.skai.accounting.service;
 
-import io.skai.accounting.dto.client.ClientRequestDto;
 import io.skai.accounting.dto.client.ClientDto;
+import io.skai.accounting.dto.client.ClientRequestDto;
 
 import java.util.List;
 
 public interface ClientService {
     ClientDto create(ClientRequestDto dto);
     List<ClientDto> getAllDto();
```

Comment:
please do not use stuff that point to type/domain of the entity in the field naming